<ui version="4.0" >
 <class>KWin::CrosshairEffectConfigForm</class>
 <widget class="QWidget" name="KWin::CrosshairEffectConfigForm" >
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Appearance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0">
       <widget class="QLabel" name="shapeLabel">
        <property name="text">
         <string>Crosshair Shape:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>shapeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="shapeComboBox">
        <property name="whatsThis">
         <string>Shape of the crosshair.</string>
        </property>
        <item>
         <property name="text">
          <string>Image</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cross</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hollow Cross</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hollow X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Square</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diamond</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="imageLabel">
        <property name="text">
         <string>Crosshair Image:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>imageKUrlRequester</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="imageKUrlRequester">
        <property name="whatsThis">
         <string>Image to use as the crosshair.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="sizeLabel" >
        <property name="text" >
         <string>&amp;Size:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>spinSize</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="KIntSpinBox" name="spinSize" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="value" >
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="widthLabel" >
        <property name="text" >
         <string>&amp;Width:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>spinWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="KIntSpinBox" name="spinWidth" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="colorLabel" >
        <property name="text" >
         <string>&amp;Color:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>comboColors</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="KColorCombo" name="comboColors" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="blendLabel">
        <property name="text">
         <string>Alpha Blending:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>blendComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="blendComboBox">
        <property name="whatsThis">
         <string>Alpha blending type.</string>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Opaque</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Transparent</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Black Background</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Invert</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Invert on Black Background</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Invert with Alpha</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Darken</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lighten</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Multiply</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="alphaLabel" >
        <property name="text" >
         <string>&amp;Alpha:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>spinAlpha</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="KIntSpinBox" name="spinAlpha" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="positionLabel">
        <property name="text">
         <string>Position:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>positionComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="positionComboBox">
        <property name="whatsThis">
         <string>Position of the crosshair.</string>
        </property>
        <item>
         <property name="text">
          <string>Screen Centre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Selected Window Centre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Active Window Centre</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QCheckBox" name="roundPositionCheckBox">
        <property name="toolTip">
         <string>Round crosshair coordinates to the nearest integer. The crosshair will look better, but may be up to 0.5 pixel off-centre.</string>
        </property>
        <property name="whatsThis">
         <string>Round crosshair coordinates to the nearest integer. The crosshair will look better, but may be up to 0.5 pixel off-centre.</string>
        </property>
        <property name="text">
         <string>Round Position</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" >
       <widget class="QLabel" name="offsetXLabel" >
        <property name="text" >
         <string>Offset X:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>offsetXSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1" >
       <widget class="KIntSpinBox" name="offsetXSpinBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>-1000</number>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="10" column="0" >
       <widget class="QLabel" name="offsetYLabel" >
        <property name="text" >
         <string>Offset Y:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>offsetYSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1" >
       <widget class="KIntSpinBox" name="offsetYSpinBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>-1000</number>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KWin::GlobalShortcutsEditor" native="1" name="editor" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KWin::GlobalShortcutsEditor</class>
   <extends>QWidget</extends>
   <header location="global" >kwineffects.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
